makefile

```diff
@@ -106,7 +106,10 @@ test-webui-local: webui/api-client webui/node_modules
 	cd webui && npm run test -- --watch=false --browsers=ChromeHeadless
 
 test-webui-e2e-local: webui/api-client webui/node_modules
-	cd webui && npm run e2e -- --dev-server-target= --base-url=http://localhost:4200 
+	cd webui && npm run e2e -- headed
```

Comment:
what does `headed` do? Will this work in an environment like a remote Linux server?

Reply:
`headed` will open the browser. I added it here because the local commands are mean for use in local development.

Reply:
"local" doesn't necessarily mean "headed" development -- it just means non-Cloud Build. I, for example, do my "local" development in a headless desktop environment, so this change will break the `test-webui-e2e-local` target for me.